sources/background_scripts/main.js

```diff
@@ -1 +1,25 @@
 NTT = {};
+
+NTT.notifyIds = [];
+
+NTT.notify = function(message){
+	var title = browser.runtime.getManifest().name;
+	// Do not display the same message twice by creating a notifyId
+	var notifyId = message.substring(0, 20);
```

Comment:
I am not sure it's best to use a sub-string as an identifier. If the goal is to not display the same message _text_ twice, then we should use the entire message as the identifier. If, however, the goal is to not display two messages that serve the same _purpose_, then it would be better to encode that purpose as a unique integer and have the caller supply it as a parameter. 

We can make it easier to ensure no two callers use the same identifier for different purposes by having these identifiers generated once through a simple provider function, something like:

```javascript
const WALLPAPER_NOTIFICATION_ID = NTT.Notification.generate_id();
```